vscode/package.json

```diff
@@ -469,7 +469,22 @@
               }
             }
           },
-          "attach": {}
+          "attach": {
+            "properties": {
+              "debugSocketPath": {
+                "type": "string",
+                "description": "The path to the debug socket. This is used to connect to the debugger"
+              },
+              "debugPort": {
+                "type": "number",
+                "description": "The port to use to connect to the debugger"
+              },
+              "debugHost": {
+                "type": "string",
+                "description": "The host to use to connect to the debugger"
+              }
```

Comment:
Can we combine these two into a single URI? I think it makes it even simpler for the user and we can easily split the host and port by using VS Code's URI API:

```typescript
const uri = vscode.Uri.parse(session.configuration.debugUri)
uri.host
uri.port
```